Automatic installation of plugin packs (#1449)
centreon/centreon-documentation: versioned_docs/version-22.04/monitoring/pluginpacks.md

### Updating the plugins

> On page **Configuration > Plugin Packs**, if **Automatic installation of plugins** is set to **ON**, plugins will be automatically updated when you export the configuration for a poller that monitors a host that uses these plugins.

Reviewer: ```suggestion
On page **Configuration > Plugin Packs**, if **Automatic installation of plugins** is set to **ON**, plugins will be automatically updated as you deploy the configuration for a poller that monitors a host that uses these plugins.
```
